lib/parse.js

```diff
@@ -68,7 +68,7 @@ module.exports = (input) => {
     device = _.chain(device)
       .split('\n')
       .filter((line) => {
-        return /^(\s\s-\s)?[a-z]+:/g.test(line);
+        return /^(\s\s[-\s]\s)?[a-zA-Z]+:/g.test(line);
```

Comment:
Why does it filter out the line? Don't you want the volumeName to also make it into the JSON objects?